scripts/deploy.sh

```diff
@@ -153,6 +157,10 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     run_ansible setup-hosts.yml --skip-tags=V-38660
   fi
 
+  # The containers must only have the base Ubuntu repository configured.
+  # All updates (security and otherwise) must come from the RPC-O apt artifacting.
+  run_ansible ${RPCD_DIR}/playbooks/configure-apt-sources.yml -e apt_target_group=all_containers
```

Comment:
Why do we need this? 
If the container was properly cleaned up (which is what we should do), we shouldn't have to enforce it.

lxc_hosts is the place we copy the apt sources from host for the default container, so if we have our artifacted default that is clean, or any other clean artifacts, it should be alright.
Less lines in deploy.sh the better.